src/Application.njs

```diff
@@ -43,6 +52,12 @@ class Application extends Nullstack {
         <button onclick={this.echoTest}> Click here to web Alert </button>
         <br></br><br></br><br></br><br></br>
         <button onclick={this.subscribe}> Click here to subscribe </button>
+        <br></br>
+        <div>
+        <input type="radio" id="monthly" name="subscribe_month" value="MONTHLY" checked="true"> Monthly </input>
+        <br></br>
+        <input type="radio" id="yearly" name="subscribe_year" value="YEARLY"> Yearly </input>
```

Comment:
the name must be the same in both input tag otherwise you are able to select both radio buttons.

Reply:
Does it? Didn't knew that, fixing